packages/main/src/components/DynamicPage/DynamicPageUtils.ts

```diff
@@ -0,0 +1,4 @@
+import { Children, ReactNode } from 'react';
+
+export const safeGetChildrenArray = <T = Exclude<ReactNode, boolean | null | undefined>>(children: unknown): T[] =>
```

Comment:
seems to be duplicated in ObjectPage. Maybe move this to some global utils and reuse it in DynamicPage as well as in ObjectPage?